services/provider/server/server_test.go

```diff
@@ -91,6 +92,22 @@ var mockExtR = map[string]*externalResource{
 			"QuotaForConsumer": fmt.Sprintf("%+v\n", clusterResourceQuotaSpec),
 		},
 	},
+	"noobaa-remote-join-secret-consumer": {
+		Name: "noobaa-remote-join-secret-consumer",
+		Kind: "Secret",
+		Data: map[string]string{
+			"auth_token": "asdasdadadadada",
```

Comment:
the value for auth_token is random?

Reply:
Just for testing purposes. Unit tests dnt care about the actual value. But the presence of it